### ISSUE TITLE
"warning: comparison of unsigned expression < 0 is always false" in PerformNameRawtx

### ISSUE THREAD

**Issue by yanmaani** [namecoin/namecoin-core#425]
Error was introduced in c145682bcae867011237866fdbf9e6e36956e816:

```
void
PerformNameRawtx (const unsigned nOut, const UniValue& nameOp,
                  CMutableTransaction& mtx, UniValue& result)
{
  mtx.SetNamecoin ();

  if (nOut < 0 || nOut >= mtx.vout.size ())
    throw JSONRPCError (RPC_INVALID_PARAMETER, "vout is out of range");
```

Since `nOut` is unsigned, the compiler warns that the first check is not needed.

Fix seems easy, but I'm not familiar with this part of the codebase.

**Comment by domob1812** [comment]
Indeed, I'll take care of it when doing the next upstream merge.

**Comment by domob1812** [comment]
Fixed in 041d115e82a1f4863f32ab8b77fcccbf7c48dbc9.
